pkg/cmd/builder.go

```diff
@@ -110,7 +118,15 @@ func (b *AdapterBase) InstallFlags() {
 			"kubeconfig file pointing at the 'core' kubernetes server with enough rights to list "+
 				"any described objects")
 		b.FlagSet.DurationVar(&b.DiscoveryInterval, "discovery-interval", b.DiscoveryInterval,
-			"interval at which to refresh API discovery information")
+			"Interval at which to refresh API discovery information")
+		b.FlagSet.Float32Var(&b.ClientQPS, "client-qps", rest.DefaultQPS, "Maximum QPS for client-side throttle")
+		b.FlagSet.IntVar(&b.ClientBurst, "client-burst", rest.DefaultBurst, "Maximum QPS burst for client-side throttle")
+
+		if b.MarkQPSFlagHidden {
+			// ignore the error if flags does not exist
+			_ = b.FlagSet.MarkHidden("client-qps")
+			_ = b.FlagSet.MarkHidden("client-burst")
+		}
```

Comment:
Now that I am seeing it, I feel like we should leave that up to the consumers of the library since at the library level we should always want the QPS flags to be created. Also, in the future we would prefer if our users switched to the new flags instead of their custom ones. wdyt @olivierlemasle?

Reply:
I'm fine with adding `ClientQPS` and `ClientBurst` and let users hide the flags if necessary (so no `MarkQPSFlagHidden`).

I raised the [question](https://github.com/kubernetes-sigs/custom-metrics-apiserver/pull/121#issuecomment-1332567134) just to make sure we're aware of this possible flag duplication, but as long as we document the change, I do not see it as an issue.

Reply:
No problem. I have deleted this part of the code. Need I add some documentation in [getting-started.md](https://github.com/kubernetes-sigs/custom-metrics-apiserver/blob/master/docs/getting-started.md#writing-the-setup-code)?